File: test/test_camera.cpp
#include <stdio.h>
#include <VideoPort.h>
#include <Camera.h>
#include <Viewer.h>
using namespace LiveKit;

int main()
{	
	VideoPort port;

	Camera cam;
	cam.AddTarget(&port);

	Viewer viewer(640, 480, "Test Camera");
	viewer.SetSource(&port);

	while (viewer.Draw());

	return 0;
}

File: python/test/test_camera.py
import LiveKit as lk

lst = lk.CameraList()
for i in range(lst.size()):
	print(lst.get(i))

port = lk.VideoPort()
cam = lk.Camera(0)
cam.add_target(port)
viewer = lk.Viewer(640, 480, "Test Camera")
viewer.set_source(port);
while viewer.draw():
	pass


